email_alert_service/models/priority.rb

```diff
@@ -0,0 +1,33 @@
+class Priority
+  def initialize(document:)
+    @document = document
+  end
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def call
+    if high_priority?
+      "high"
+    else
+      "normal"
+    end
+  end
+
+private
+
+  attr_reader :document
+
+  def travel_advice?
+    document["document_type"] == "travel_advice"
+  end
+
+  def medical_safety_alert?
+    document["document_type"] == "medical_safety_alert"
+  end
+
+  def high_priority?
+    travel_advice? || medical_safety_alert?
```

Comment:
I'd maybe consider ditching the separate methods and just have %w[medical_safety_alert travel_advice].include?(document["document_type"])

Could even have a constant